src/Compilers/CSharp/Portable/Symbols/Tuples/TupleTypeSymbol.cs

```diff
@@ -1468,7 +1468,7 @@ internal override IEnumerable<CSharpAttributeData> GetCustomAttributesToEmit(Mod
 
         public static TypeSymbol TransformToTupleIfCompatible(TypeSymbol target)
         {
-            if (target.IsTupleCompatible())
+            if (target.IsTupleCompatible() && !target.IsErrorType())
```

Comment:
Alternatively, should we just fold this into `IsTupleCompatible`? I can see a solid justification that ErrorTypes are not tuple compatible.